spec/unit/optmath_spec.lua

```diff
@@ -0,0 +1,49 @@
+describe("Math module", function()
+
+    setup(function()
+        require("commonrequire")
+        Math = require("optmath")
+    end)
+
+    it("should round away from zero", function()
+        local num = 1.5
```

Comment:
I suppose it's mostly the extra linebreak. Anyway it's not like it matters. I'll get rid of the vars